examples/docker-swarm-mode/converge-hcl/main.hcl

```diff
@@ -0,0 +1,11 @@
+param "docker-group-user-name" {}
```

Comment:
now I remember why I had named it this way. In the docker.hcl module it is just `user-name` but I decided to namespace it in the top-level module. Since we don't actually have a naming conflict right now, I will go back and rename it to just `user-name`.